kie-wb-tests/kie-wb-tests-gui/src/main/java/org/kie/wb/selenium/model/Persp.java

```diff
@@ -92,7 +92,7 @@
 
     private static final List<Persp<? extends AbstractPerspective>> ALL_PERSPS = Collections.unmodifiableList(Arrays.asList(
             HOME_PAGE, TIMELINE, PEOPLE, PROJECT_AUTHORING, CONTRIBUTORS, ARTIFACT_REPOSITORY, ADMINISTRATION,
-            PROCESS_DEPLOYMENTS, RULE_DEPLOYMENTS, JOBS, PROCESS_DEFINITIONS, PROCESS_INSTANCES, TASKS,
+            RULE_DEPLOYMENTS, JOBS, PROCESS_DEFINITIONS, PROCESS_INSTANCES, TASKS,
```

Comment:
A minor comment: rename RULE_DEPLOYMENTS constant name to EXECUTION_SERVERS to make it consistent with the new perspective name.